python/tk_hiero_export/sg_audio_export.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, initDict):
         self._sequence_name = None
         self._thumbnail = None
 
+        # Retrieving the bithDepth and bitRate values
+        self._bitDepth = initDict["preset"]._properties["bitDepth"]
+        self._bitRate = initDict["preset"]._properties["bitRate"]
```

Comment:
Can you confirm if these properties are available in all supported versions of nuke?

Reply:
[here](https://github.com/shotgunsoftware/tk-hiero-export/pull/54/files#r792213394)